Song_sort_Genre.css

```diff
@@ -1,64 +1,38 @@
+/* Import  font from google fonts*/
```

Comment:
21/9: I import font from google fonts. The fonts I choice is Source sans pro and Open sans